src/main.ts

```diff
@@ -34,13 +34,23 @@ function getGitHubToken(): string {
     return token;
 }
 
-async function getCargoSemverChecksDownloadURL(target: string): Promise<string> {
+async function getCargoSemverChecksDownloadURL(target: string, version: 'latest' | string): Promise<string> {
     const octokit = github.getOctokit(getGitHubToken());
 
-    const getReleaseUrl = await octokit.rest.repos.getLatestRelease({
-        owner: "obi1kenobi",
-        repo: "cargo-semver-checks",
-    });
+    let getReleaseUrl;
+
+    if (version === 'latest') {
+        getReleaseUrl = await octokit.rest.repos.getLatestRelease({
+            owner: "obi1kenobi",
+            repo: "cargo-semver-checks",
+        });
+    } else {
+        getReleaseUrl = octokit.rest.repos.getReleaseByTag({
+            owner: "obi1kenobi",
+            repo: "cargo-semver-checks",
+            tag: version,
+        });
```

Comment:
Could you make sure the error message if the user supplied an invalid tag is good and gives the user clear and actionable feedback on what happened and how to fix it?